vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SequenceManagerTest.java

```diff
@@ -0,0 +1,90 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.diging.vspace.core.data.SequenceRepository;
+import edu.asu.diging.vspace.core.factory.impl.SequenceFactory;
+import edu.asu.diging.vspace.core.model.ISequence;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.impl.Module;
+import edu.asu.diging.vspace.core.model.impl.Sequence;
+import edu.asu.diging.vspace.web.staff.forms.SequenceForm;
+
+public class SequenceManagerTest {
+
+    @Mock
+    private SequenceRepository mockSequenceRepo;
+
+    @Mock
+    private ModuleManager moduleManager;
+    
+    @Mock
+    private SlideManager slideManager;
+    
+    @Mock
+    private SequenceFactory sequenceFactory;
+    
+    @InjectMocks
+    private SequenceManager sequenceToTest = new SequenceManager();
+    
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void test_getSequence_idExists() {
+        Sequence newSequence = new Sequence();
+        newSequence.setId("sequence1");
+        Optional<Sequence> mockSequence = Optional.of(newSequence);
+        Mockito.when(mockSequenceRepo.findById(newSequence.getId())).thenReturn(mockSequence);
+
+        ISequence iSequenceActual = sequenceToTest.getSequence(newSequence.getId());
+        Assert.assertEquals(mockSequence.get().getId(), iSequenceActual.getId());
+        Assert.assertEquals(mockSequence.get().getName(), iSequenceActual.getName());
+    }
+    
+    @Test
+    public void test_getSequence_idNotExists() throws Exception {
+        Mockito.when(mockSequenceRepo.findById(Mockito.anyString())).thenReturn(Optional.empty());
+        Assert.assertNull(sequenceToTest.getSequence(Mockito.anyString()));
```

Comment:
```Mockito.anyString()``` isn't really meant for this case and returns an empty string, which is kind of a special case. just pass an actual string.